Jenkinsfile

```diff
@@ -0,0 +1,45 @@
+pipeline {
+
+    options {
+        timestamps()
+        timeout(time: 30, unit: 'MINUTES', activity: true)
+    }
+
+    agent {
+        node {
+            label "cloud-ccp-ci"
+        }
+    }
+    environment {
+        SOCOK8S_ENVNAME = "cloud-socok8s-${env.BUILD_NUMBER}"
```

Comment:
Is this really going to be unique? Doesn't the BUILD_NUMBER start from 1 from each PR? In that case it might make sense to add `env.BRANCH_NAME` to the `SOCOK8S_ENVNAME`.

Reply:
Indeed we used BUILD_NUMBER in periodics but that was always the same job, here the jobs will all be different I guess...

Reply:
Fixed. Added `env.BRANCH_NAME`.